### DISCUSSION TITLE
I didn't receive my education benefits within 5 days of your approval status

### DISCUSSION THREAD

**Discussion by hind-kharbouche** [community/community#144416] [GitHub Education]
### Hi everyone,

Hi,

I submitted a request for GitHub Education status twice. Initially, my request was declined, but on my second submission, it was approved. However, I haven’t yet received my education benefits, and it’s been more than **five days** since the approval.

Could you please assist me with this?

Thank you!

**Comment by davevad93** [comment]
Hi @hind-kharbouche ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @hind-kharbouche was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by hind-kharbouche** [reply]
Thank you for your response!

**Reply by davevad93** [reply]
You're welcome @hind-kharbouche 👍
